util/test_base_opencl.cpp

```diff
@@ -107,6 +113,28 @@ bool test_base_opencl::online_compiler_supported(cl_device_id clDeviceId,
   return true;
 }
 
+bool test_base_opencl::get_exec_dir(char *path, size_t max_path_len) {
+#ifdef _MSC_VER
+  HMODULE hMod = GetModuleHandle(NULL);
+  if (!GetModuleFileNameA(hMod, path, max_path_len)) {
+    return false;
+  }
+#else
+  ssize_t n = readlink("/proc/self/exe", path, max_path_len - 1);
+  if (n < 0) {
+    return false;
+  }
+  path[n] = '\0';
+#endif
+  // Replace all characters with '\0' from back until '/' or '\\'
+  for (size_t i = strnlen(path, max_path_len) - 1;
+       i > 0 && path[i] != '/' && path[i] != '\\'; --i) {
```

Comment:
You have to decide whether you are on Windows or not here too.
For example this does not work on Unix if you have `\` in the path...

Reply:
Done.